### ISSUE TITLE
update npm dependencies

### ISSUE THREAD

**Issue by ben-nathanson** [pgh-public-meetings/events#9]
There is a potential vulnerability in our npm dependencies. This is not yet patched by `react-scripts` but should be soon. More info [here](https://github.com/facebook/create-react-app/issues/9842). No action is possible to fix this yet on our end.

**Comment by ben-nathanson** [comment]
Specifically

```
                       === npm audit security report ===                        
                                                                                
┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ object-path                                                  │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=0.11.5                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ react-scripts                                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ react-scripts > resolve-url-loader > adjust-sourcemap-loader │
│               │ > object-path                                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1573                            │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 1 high severity vulnerability in 1697 scanned packages
  1 vulnerability requires manual review. See the full report for details.
```

**Comment by ben-nathanson** [comment]
This is a duplicate of #10. Closing now.
